### ISSUE TITLE
Schema relaxing?

### ISSUE THREAD

**Issue by eu-genia** [BetaMasaheft/Documentation#2197]
I was wondering after all if, in order to survive, we could relax schema in some cases.
For example, for the moment place requests a `type`, but there are many records (1700) from the EAe but also created by BM without. I could of course insert something, but we could also set the type to optional. (Am not sure whether it is worth it but wanted to have it mentioned -- place `@type` for sure was optional once, at least when i created all the entries for TraCES linking in 2017-2018 there was no forced type).
Maybe there are other cases where schema relaxation could be justified.
What do you think @thea-m @DenisNosnitsin1970

**Comment by DenisNosnitsin1970** [comment]
It depends, and probably each case  should be treated individually. On one hand, at some points schema can probably be more relaxed, on the other hand, adding categories, even preliminary, ist one of the reasons of encoding, isn't it?

**Comment by thea-m** [comment]
A parallel would be the type for occupation in persons. The problem here are the EAe inherited records. I think if we relax the schema to ease the validation task for us, that would be fine. We would keep the guidelines as they are. If we then notice that people (including us) stop using these categories in newly created or updated records, we could reconsider.

**Comment by eu-genia** [comment]
Agree. It would be enough to replace usage="req" to usage="opt" in most categories to have most of our records valid.

Sorry is the relation `dc:relation `deprecated? It used to be there and now it also is invalid.

Maybe we should also take back things that were removed, marking them, as suggested, as deprecated, this would also save us a lot of work and headache.

**Comment by thea-m** [comment]
I don't think dc:relation is deprecated. Not the first time the problem arises: https://github.com/BetaMasaheft/Documentation/issues/1343

**Comment by eu-genia** [comment]
Changes from #1343 are not in the current schema file, this is why i was wondering...

**Comment by eu-genia** [comment]
dc:relation is back in )

**Comment by eu-genia** [comment]
@thea-m do you know how to allow elements to be empty? a lot of validation errors arise from things like
`<listWit/>`
or 
`<listRelation/>`
in many of the automatically transformed records as the schema wants to have something inside.
Maybe it is not so important if things like that are allowed to be empty and be filled later?

**Comment by eu-genia** [comment]
All files are valid
